### DISCUSSION TITLE
"Solved in"  more than 100 secondes

### DISCUSSION THREAD

**Discussion by Jewem** [FlareSolverr/FlareSolverr#1151] [General]
Good evening, I'm using Flaresolverr on my dedicated server exclusively for the YGGTorrent tracker. 
I believe I've checked all my configurations, but I'm still unable to achieve stability. Flaresolverr solves the challenge each time, but in times that are really too long. 
Jackett therefore regularly errors out for this tracker because the response time exceeds 100 seconds. 

 _""An error occurred while testing this indexer
Exception (yggtorrent): A task was canceled.: The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.""_

I've extracted data from my logs to share the server's performance with you. You'll notice that the average response time is 136 seconds, and there are few instances where I'm below 100. Can you please help me? I'm at a loss for what to do."

![image](https://github.com/FlareSolverr/FlareSolverr/assets/150742202/e1b69c54-ea22-4b50-9fa7-f339af3c5972)

**Data since 6 april 2024  
100 seconde deadling in green
Average in yellow**

**Comment by Jewem** [comment]
Out of 212 challenges resolved since April 6th, I have a response rate of less than 10 seconds for 44% of them.

**Comment by ilike2burnthing** [comment]
Unless the host device is underpowered (e.g. a cheap Pi/SBC), or resources are being limited to the container, see https://github.com/FlareSolverr/FlareSolverr/issues/1036.
